woocommerce/woocommerce
# Merge wc-wp-version-gte-53 rules to the main CSS

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





We currently [require at least 6.5](https://github.com/woocommerce/woocommerce/blob/5c3236692ca35b3335c3e1d5b459feae1ece3bd2/plugins/woocommerce/woocommerce.php#L11:L11) Therefore, we can safely assume WP is greater than 5.5 and can remove some compatibility code for earlier versions. This PR does not aim to remove all compact code, only merge CSS rules and logic around `wc-wp-version-gte-*`. However, it keeps adding those classes to the body, as there are some extensions that still use those.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

This PR does not introduce or change any functionality. It's a codebase cleanup to remove dead code.
So we are mostly testing for regression.

1. Checkout and build this branch
2. Check the font size and cursor over help tips for example at Edit order page
3. Check the margin top of the `::after` element (icon) of a action column button. For example at Orders list table `/wp-admin/admin.php?page=wc-orders`, you can search for `document.querySelectorAll('.widefat .column-wc_actions a.button')` and make them visible, by removing `hidden` class from the parent `td`
4. Check the "REST API" header margin at `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=keys`
5. Check padding of form table inputs for example at `/wp-admin/admin.php?page=wc-settings&tab=general`
	- Check their width when you make the screen smaller than 782px
6. Check the margins and positioning of help tips in form tables, for example at `/wp-admin/admin.php?page=wc-settings&tab=general`
7. Check the fontsize of input color in form table for example at `/wp-admin/admin.php?page=wc-settings&tab=email`
8. Check the padding, dimensions, fontsize, margins and borders of color picker previews, for example at `/wp-admin/admin.php?page=wc-settings&tab=email`, check its dimensions when you resize to < 782px.
9. Check the border, shadow ad positioning of elements of select2 dropdowns, like the ones at `/wp-admin/admin.php?page=wc-settings&tab=email&section=wc_email_new_order`
10. Go to product or coupon editor 
	1. check that tabs in product-data panel are displayed correctly even without `.wc-tabs-back` element.
	2. check that type selector is added correctly to the prodcut-data header


I didn't find the occurrence of `.woocommerce input + .subsubsub`



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
